### ISSUE TITLE
[Generated by cli] Potential redudant packages that are not covered by nolyfill

### ISSUE THREAD

**Issue by alex-kinokon** [SukkaW/nolyfill#57]
### `call-bind`

```
$ pnpm why call-bind
```

**stdout**

```
Legend: production dependency, optional only, dev only

devDependencies:
patch-package 8.0.0
└─┬ json-stable-stringify 1.1.0
  └── call-bind 1.0.5

```

**Comment by SukkaW** [comment]
The target package is `json-stable-stringify`.

**Comment by SukkaW** [comment]
Closing the issue now as the `@nolyfill/json-stable-stringify` is already released in nolyfill 1.0.30.
